notebooks/instructlab-knowledge/level1.ipynb

```diff
@@ -0,0 +1,1016 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "id": "5da70a08-1895-4d1f-8f50-93e2134b2e23",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# design goals:\n",
+    "#\n",
+    "# - understandability\n",
+    "# - modularity\n",
+    "# - configurability"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "id": "0acd026f-65bd-4393-bb40-f8aa8bd6828b",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from pathlib import Path\n",
+    "\n",
+    "WORKSPACE_NAME = \"default\"\n",
+    "\n",
+    "WORKSPACE_ROOT = Path(\"workspaces\")\n",
+    "WORKSPACE_ROOT.mkdir(exist_ok=True)\n",
+    "\n",
+    "WORKSPACE_DIR = WORKSPACE_ROOT / WORKSPACE_NAME\n",
+    "WORKSPACE_DIR.mkdir(exist_ok=True)\n",
+    "\n",
+    "SOURCE_DOCUMENT = None # to process a specific document, set its path here; otherwise, the entire source documents repository will be used\n",
```

Comment:
Is there a "call to action" we can provide or way to make it extremely obvious to the user that they need to set their source doc here?

Reply:
I had a similar observation. I moved this to its own cell in my PR, but I'm open to other ways to handle this

Reply:
And that change was merged - thanks!